source/acquaman/AMGenericScanActionControllerAssembler.cpp

```diff
@@ -124,16 +124,28 @@ AMAction3* AMGenericScanActionControllerAssembler::generateActionTreeForStepAxis
 		// generate axis initialization list
 		AMListAction3 *initializationActions = new AMListAction3(new AMListActionInfo3(QString("Initializing %1").arg(axisControl->name()), QString("Initializing Axis with Control %1").arg(axisControl->name())), AMListAction3::Sequential);
 
-		AMAction3 *initializeControlPosition = AMActionSupport::buildControlMoveAction(axisControl, stepScanAxis->axisStart());
+		AMAction3 *initializeControlPosition = 0;
+
+		if (direction_ == AMScanConfiguration::Increase)
+			initializeControlPosition = AMActionSupport::buildControlMoveAction(axisControl, stepScanAxis->axisStart());
```

Comment:
Just curious, because I'm not sure exactly how directions work for scan axes. Does the `axisStart()` method take into account the desired scan direction, or is it the initial position of the first region?